### ISSUE TITLE
Should be possible to install in a subdirectory

### ISSUE THREAD

**Issue by nrnrnr** [daurnimator/lamt#2]
Grabbing module names like 'general', 'misc', and 'id3v2' is probably not a good plan.  It would be good if it were possible to install the package as, say, 'lamt.id3v2' and so on.

See, for example, https://stackoverflow.com/questions/9145432/load-lua-files-by-relative-path

**Comment by daurnimator** [comment]
Indeed. This project was originally written as a tool rather than as a library; but being a library would be very sensible!

Unfortunately, I haven't had time for this project in quite a few years, and it doesn't look like I'll have any more time coming up.

If you'd like to make contributions, I'm happy to review and merge.

**Comment by nrnrnr** [comment]
You're very kind.  I had a harder look at the code last night, and I think without a little more API documentation I'm likely to struggle.  I may see what I can squeeze out of mutagen, but it's nice to have this as a possible backup.
